tests/integration/test_prepare_tools.py

```diff
@@ -56,6 +56,21 @@ def test_volume_extract_from_edge_loops(modeler: Modeler):
     assert len(created_bodies) == 1
 
 
+def test_remove_rounds(modeler: Modeler):
+    """Test a round is removed from the geometry."""
+    from ansys.geometry.core.designer import SurfaceType
+
+    design = modeler.open_file(r"C:\temp\BoxWithRound.scdocx")
```

Comment:
@smereu looks like file path needs to be updated.

Reply:
Very good catch @umutsoysalansys!! I completely missed that one